test/library/standard/Path/dlongnecke/relPath/fileRelPath.preexec

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 
-# Produces the "fileRelPath.good" file containing expected output for the
-# Chapel file.absPath function.
+# Produces the output file containing expected output for the Chapel
+# file.absPath function.
```

Comment:
This still says file.absPath :)

Reply:
Less documentation is better I see... xD